.github/workflows/ci.yml

```diff
@@ -155,58 +155,53 @@ jobs:
         with:
           file: coverage/lcov.info
 
-  build-deploy-frontend:
+  build-nextjs-container:
```

Comment:
I'd be happy to just remove this step—we have `yarn workspace main build` above. (Which, annoyingly, we do need to run separately. It has to run before typecheck. https://github.com/vercel/next.js/issues/53959#issuecomment-1735563224)

If you want to keep it, I would remove the heroku vars.

Reply:
The intention is to catch issues building the container before that happens on the release branches, but:
- As you say, we end up doing 2 builds on push (albeit in parallel).
- It's not the exact same as release branches as there we build using Heroku's CLI [`heroku container:push`](https://github.com/heroku/cli/blob/main/docs/container.md#heroku-containerpush) (they don't have a build only / dry run command).

It would be good to have some check on the docker build before we get to `release-candidate`. Can we rethink once this is released?  Good call with Heroku vars - removed.